modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -284,31 +201,86 @@ Runtime Fabric is not deployed behind a load balancer, these values should not b
+
+=== External Load Balancer Requirements
+When running multiple controller VMs, you must have an external load balancer outside Runtime Fabric to front each of the controller VMs.
+
+The external load balancer must support TCP load balancing and must be configured with a server pool containing the IP addresses of each controller VM. A health check must also be configured on the external load balancer, listening on port 443.
+
+This configuration of the external load balancer provides the following benefits:
+
+* Maintains high availability.
+* Protects against failures.
+* Gracefully handles automated failover if a replica of the internal load balancer restarts or is evicted and rescheduled on another controller VM.
+
+To configure an external load balancer:
+
+. Review the information described in *Advanced Options* when adding an external load balancer.
+. Configure DNS before using the CN obtained from the TLS certificate. DNS is required to send requests to applications or API gateways deployed to Runtime Fabric. Add an "A record" to your DNS provider to map the CN to the IP address of the external load balancer or controller VMs.
 
-* FATAL
-* ERROR
-* WARNING
-* INFO
-* VERBOSE
-* DEBUG
-* TRACE
+== Step 5: Deploy Applications
+When you are ready to deploy an application, follow the instructions in xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric].
 
-The more verbose log levels, WARNING to TRACE, consume more CPU resources for each request. Consider this when adjusting the log level and allocating resources for the internal load balancer.
+The *Ingress* tab allows you to specify whether inbound traffic should be allowed for the application. After you enable inbound traffic, the default behavior is changed to allow for new application deployments. If there are applications deployed to Runtime Fabric before you enable inbound traffic, they do not receive inbound requests until this setting is enabled.
 
-By default, the activity across all IPs addresses behind your endpoint are logged. To help reduce CPU consumption when using more verbose log levels, add IP filters to only log specific IP addresses. This feature also reduces the quantity of logs when debugging a connection for a specific or limited number of IP addresses.
+== TLS Certificate Expiration
+Certificates (both self-signed and CA-signed) always have an expiration date. By default, certificates expire one year after they are created. 
 
-=== Configuring Internal Load Balancer Logs
+The following warnings are displayed for certificates that will expire within 30 days to allow you to upload a new certificate-key pair before a certificate expires:
```

Comment:
```suggestion
The following warnings are displayed for certificates that will expire within 30 days so that you remember to upload a new certificate-key pair before a certificate expires:
```